src/SFA.DAS.LoginService.Web/Views/Shared/_Layout.cshtml

```diff
@@ -157,13 +157,13 @@
                     <h2 class="govuk-visually-hidden">Support links</h2>
                     <ul class="govuk-footer__inline-list">
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" asp-controller="Home" asp-action="TermsOfUse" asp-route-clientId="@ViewBag.ClientId">Terms of use</a>
+                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Cookies" asp-route-clientId="@ViewBag.ClientId" target="_blank">Cookies</a>
```

Comment:
SonarCloud suggests adding `noopener` and `noreferrer` relations to links that open in a new tab (related to [this security concern](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/))

```suggestion
                            <a class="govuk-footer__link" asp-controller="Home" asp-action="Cookies" asp-route-clientId="@ViewBag.ClientId" target="_blank" rel="noopener noreferrer">Cookies</a>
```

Reply:
Thanks for this, I've amended the links